group_vars/betabarrel_cluster/ip_addresses.yml

```diff
@@ -1,7 +1,10 @@
 ---
 ip_addresses:
   betabarrel:
-    vlan983:
-      address: 172.23.41.225
+    bb_internal_management:
+      address: 129.125.55.13
```

Comment:
```suggestion
    vlan983:
      address: 172.23.41.225
    vlan13:
      address: 129.125.55.13
```

Reply:
I updated this part with new commit.
I fixed the naming and also added the correct netmasks on both IPs.